src/pages/Games/Games.tsx

```diff
@@ -1,10 +1,28 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useFirestoreCollection } from '@/hooks/useFirestoreCollection';
```

Comment:
Combine all hooks in one `index.ts` file and import if needed as following: `import {...} from @/hooks`